src/ja/server/proxy/command_handler.py

```diff
@@ -60,9 +64,14 @@ def _user_is_admin(self, user: str) -> bool:
         return self._admin_group in groups
 
     def _execute_command(self, command: ServerCommand) -> Dict[str, object]:
-        return command.execute(self._database).to_dict()
+        logger.info("executing server command")
+        logger.debug("%s" % str(command))
+        r_dict = command.execute(self._database)
+        logger.debug("%s" % str(r_dict))
```

Comment:
same

Reply:
Also, you write something like `"Response: %s" % str(r_dict)` to be clear that you're printing the response.